maestro/service/deploy_service.go

```diff
@@ -106,23 +106,14 @@ func (d *eventService) HandleSinkActivity(ctx context.Context, event maestroredi
 		return errors.New("trying to deploy sink that is not active")
 	}
 	d.logger.Debug("handling sink activity event", zap.String("sink-id", event.SinkID))
-	deploymentEntry, _, err := d.deploymentService.GetDeployment(ctx, event.OwnerID, event.SinkID)
```

Comment:
this is correct, activity should ignore the errored sinks/ collectors

Reply:
reverted